src/Service/Transifex/ResourcesService.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Service\Transifex;
+
+use App\Service\Transifex\DTO\ResourceDTO;
+use Mautic\Transifex\Connector\Resources;
+use Mautic\Transifex\Exception\ResponseException;
+use Mautic\Transifex\TransifexInterface;
+use Symfony\Component\Console\Logger\ConsoleLogger;
+
+class ResourcesService
+{
+    public function __construct(
+        private readonly TransifexInterface $transifex,
+        private readonly LanguageStatsService $languageStatsService
+    ) {
+    }
+
+    public function processAllResources(ResourceDTO $resourceDTO, ConsoleLogger $logger): int
+    {
+        try {
+            $resources    = $this->transifex->getConnector(Resources::class);
+            $response     = $resources->getAll();
+            $body         = json_decode($response->getBody()->__toString(), true);
```

Comment:
```suggestion
            $body         = json_decode((string) $response->getBody(), true);
```
I would prefer type casting to calling magic methods explicitly.